lib/ansible/modules/network/cloudengine/ce_vrf_interface.py

```diff
@@ -0,0 +1,517 @@
+#!/usr/bin/python
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'metadata_version': '1.0'}
+
+DOCUMENTATION = '''
+---
+module: ce_vrf_interface
+version_added: "2.4"
+short_description: Manages interface specific VPN configuration on HUAWEI CloudEngine switches.
+description:
+    - Manages interface specific VPN configuration of HUAWEI CloudEngine switches.
+author: Zhijin Zhou (@CloudEngine-Ansible)
+notes:
+    - Ensure that a VPN instance has been created and the IPv4 address family has been enabled for the VPN instance.
+options:
+    vrf:
+        description:
+            - VPN instance, the length of vrf name is 1 ~ 31,i.e. "test", but can not be _public_.
```

Comment:
`VPN instance, the length of vrf name is 1 ~ 31, i.e. "test", but can not be C(_public_.)`